concepts/05 Widgets/Tabs/00 Overview.md

```diff
@@ -15,11 +18,90 @@ The following code adds the **Tabs** widget to your page. One of the tabs has an
             items: [
                 { text: "User", icon: 'user' },
                 { text: "Comment", badge: "New" },
-                { text: "Find" }
+                { text: "Find", disabled: true }
             ]
         });
     });
 
+##### Angular
+
+    <!--HTML-->
+    <dx-tabs
+        [items]="tabs">
+    </dx-tabs>
+
+    <!--TypeScript-->
+    import { DxTabsModule } from "devextreme-angular";
+    // ...
+    export class AppComponent {
+        tabs = [
+            { text: "User", icon: 'user' },
+            { text: "Comment", badge: "New" },
+            { text: "Find", disabled: true }
+        ];
+    }
+    @NgModule({
+        imports: [
+            // ...
+            DxTabsModule
+        ],
+        // ...
+    })
+
+##### Vue
+
+    <!--tab: App.vue-->
+    <template>
+        <DxTabs
+            :items="tabs" />
+    </template>
+    <script>
+    import DxTabs from "devextreme-vue/tabs";
+
+    export default {
+        components: {
+            DxTabs
+        },
+        data() {
+            return {
+                tabs: [
+                    { text: "User", icon: 'user' },
+                    { text: "Comment", badge: "New" },
+                    { text: "Find", disabled: true }
+                ]
+            };
+        }
+    };
+    </script>
+
+##### React
+
+    <!--tab: App.js-->
+    import React from 'react';
+    import { Tabs } from 'devextreme-react/tabs';
+
+    class App extends React.Component {
+        constructor() {
```

Comment:
You don't need a constructor to declare an array; it can be declared right in the component. Anyway, since the tabs aren't going to be changed, I'd suggest moving them to a constant before the class declaration.